scripts/create_newcase

```diff
@@ -12,7 +12,7 @@ def parse_command_line(args, cimeroot, description):
 ###############################################################################
     help_str = \
 """
-%s --case [CASE] --compset [COMPSET] --res [GRID] [--mach ...] [--compiler ...]
+%s --case [CASE] --compset [COMPSET] --res [GRID] [--machine ...] [--compiler ...]
```

Comment:
No we kept -mach for backward compatibility, I don't think that --mach has had much use.